tests/unit/.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+  'extends': '../../.eslintrc',
+
+  'env': {
+    'mocha': true
+  },
+
+  'globals': {
+    'expect': false,
+    'sandbox': false,
+    'document': false,
+    'window': false,
+    'React': false,
+    'ReactDOM': false,
```

Comment:
`React`, `ReactDOM` are not globals. And `TestUtils` is not even used (at least not in this PR).